CONTRIBUTING.md

```diff
@@ -99,21 +99,23 @@ Further reading:
     1. Make sure your commit message follows the formatting requirements (details above) and here: [Conventional Commits syntax](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification); this aids in release notes generation which we intend to automate
     2. Be aware that we are using git commit hooks for the automation of certain mundane tasks such as applying the required code style and formatting so your code will be wrapped at 80 characters each line automatically. If you wish to see how your changes will be altered by the formatter you can run the `npm run prettier` command from a terminal or install an IDE extension for the `Prettier` tool that can do the same (VSCode has one that is known to work).
 8. Ensure your branch is rebased onto the `upstream` main branch where `upstream` is fancy git talk for the main Cactus repo on Github (the one you created your fork from).
-   1. If you are having trouble, there are many great resources out there (so we will not write another here).
+   1. **Do not** duplicate your pull request after it has been reviewed. Duplication here means closing the existing PR and then opening a brand new one which does not contain the review history anymore. If you encounter issues with version control that you do not know how to solve the maintainers will be happy to assist to ensure that you do not need to open a new pull request from scratch.
```

Comment:
@Leeyoungone Please double check if these are changes leftover from a rebase/conflict resolution gone sideways. If you know for a fact that there's no changes in this file anymore that you wanted to make in this PR (because all of it is now in the BUILD.md file) then you can also just restore CONTRIBUTING.md to whatever state it is currently in on the main branch.